service/cleanup.go

```diff
@@ -0,0 +1,28 @@
+package service
+
+import (
+	"fmt"
+	"gorm.io/gorm"
+	"mce.salesforce.com/sprinkler/database"
+	"mce.salesforce.com/sprinkler/database/table"
+	"time"
+)
+
+type Cleanup struct {
+	ScheduledWorkflowTimeout time.Duration
+}
+
+func (s *Cleanup) Run() {
+	fmt.Println("Cleanup started")
+	s.deleteExpiredScheduledWorkflows(database.GetInstance())
+	fmt.Println("Cleanup complete")
+}
+
+func (s *Cleanup) deleteExpiredScheduledWorkflows(db *gorm.DB) {
+	expiryTime := time.Now().Add(-s.ScheduledWorkflowTimeout)
+
+	db.Unscoped().
```

Comment:
Added Unscoped to cause hard deletion of records in a table with no primary key. See https://gorm.io/docs/delete.html#Delete-permanently